x-pack/plugins/security_solution/server/endpoint/services/actions/clients/sentinelone/sentinel_one_actions_client.ts

```diff
@@ -178,13 +182,14 @@ export class SentinelOneActionsClient extends ResponseActionsClientImpl {
     }
   }
 
-  async isolate(options: IsolationRouteRequestBody): Promise<ActionDetails> {
+  private async handleResponseAction(command: SUB_ACTION, options: IsolationRouteRequestBody) {
```

Comment:
Just FYI: I was waiting for us to include more involved response actions before doing this because I don't know that centralizing it for an external EDR makes sense yet - there may not be consistency in how all response actions are handled for those system and each response action may need to have its own logic.